pyproject.toml

```diff
@@ -0,0 +1,17 @@
+[project]
+name = "pypolyclip"
+description = "A python driver for polyclip written in C by JD Smith."
+dynamic = [
+    "version"
+]
```

Comment:
This project information is duplicated in `setup.cfg`.  Ideally, it should go in one or the other.

Reply:
Stripped duplicated info in https://github.com/spacetelescope/pypolyclip/pull/4/commits/25d2bfa992328e184bc3f102369bf7e9f94b1ad9